spec/lib/local-links-manager/import/links_importer_spec.rb

```diff
@@ -104,6 +104,42 @@
 
         expect(link.reload.url).to eq 'http://www.example.com/this-is-now-different'
       end
+
+      context 'when a link is missing from the CSV' do
+        it 'is removed from the database' do
+          service = FactoryGirl.create(:service, lgsl_code: 123, label: 'Service 123')
```

Comment:
It'd certainly help get some of the boilerplate out of the tests, so it sounds like a reasonable thing to me.